src/components/Input/FileInput.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ChangeEvent } from 'react';
+import Input from './Input';
+
+type FileInputProps = {
+  onChange: (files: FileList | null) => void;
+  type: 'file';
+};
+
+class FileInput extends React.Component<FileInputProps> {
```

Comment:
Can we convert this to a functional component as well?

Reply:
Yes 👍